CONTRIBUTING.md

```diff
@@ -45,6 +45,11 @@ What constitutes a business day is managed by the gem `business_time` and is con
 
 ### App setup
 
+Before running bin/setup, ensure that 'foreman' is removed from the Gemfile.  
```

Comment:
This part of the setup confused us.  There's a line in the App Setup section that says:

> "If you don't have foreman, see Foreman's install instructions. It is _purposefully excluded_ from the project's Gemfile." 

It says that the Foreman gem is excluded, but it's not - it's actually still in the Gemfile (line 10).  This seemingly conflicting information confused us during the setup process (we are all pretty inexperienced devs).  Again, I'll have to defer to you on how to best address this issue - all I can really do is say that this was a pain point for us during setup.